### ISSUE TITLE
strange terminal character output

### ISSUE THREAD

**Issue by jmcneese** [FormidableLabs/nodejs-dashboard#12]
added import to my server entrypoint:
`import 'nodejs-dashboard';`

added npm script like so:
`"start:dev": "NODE_ENV=development nodemon --watch server --exec \"npm run build:schema && nodejs-dashboard babel-node server/index.js\""`

get these escape characters after running for a minute or so: 
<img width="1436" alt="screenshot 2016-10-12 22 26 43" src="https://cloud.githubusercontent.com/assets/49743/19336843/ef3dedb4-90cb-11e6-882f-91ffc3b41c0f.png">

this only occurs when my terminal is in rxvt mode, not in ansi mode.

**Comment by joshontheweb** [comment]
I have had a similar issue running 
`forever -c "nodejs-dashboard node" server.js`

It just spits out the escape characters when I move the mouse but I don't even get to the dashboard view.

It seems to work fine running without forever.

**Comment by jlc467** [comment]
Just curious, are you using iTerm2 with mouse reporting enabled? If so, try disabling and report back. It solved the issue for me.

![node-dash-issue12](https://cloud.githubusercontent.com/assets/6405557/19411031/708bea54-92c7-11e6-95cd-d9cdd2b26bfd.gif)

**Comment by vitosamson** [comment]
I was able to work around this by running nodemon inside of nodejs-dashboard: `nodejs-dashboard -- nodemon -x 'node .'`

**Comment by jasonwilson** [comment]
@jmcneese were you able to get around this issue?

**Comment by joshontheweb** [comment]
@jlc467 Yes, I am using iterm with the mouse reporting so that makes sense.

In the end this worked fine for me:
`nodejs-dashboard forever server.js`

**Comment by tanepiper** [comment]
I've tried @vitosamson's solution but it doesn't work for me fully.  The dashboard starts but the event loop, memory and cpu are not working.  Other wise I get the same character rendering issue.

I tried forever too in watch mode but couldn't get it to work, it would just crash but spawn an instance.

**Comment by p77u4n** [comment]
I have same problem :(

**Comment by jjasonclark** [comment]
Adding link to related Blessed issue https://github.com/chjj/blessed/issues/235
